package/meta.yaml

```diff
@@ -71,41 +71,41 @@ outputs:
         - cartopy>=0.18
         - cdo>=1.9.7
         - eccodes!=2.19.0  # cdo dependency; something messed up with libeccodes.so
-        - cdsapi
-        - cf-units
-        - cftime
-        - cmocean
+        - cdsapi>=0.5.1
+        - cf-units>=2.1
+        - cftime>=1.2.1
+        - cmocean>=2.0
         - dask>=2.12
         - ecmwf-api-client  # in esmvalgroup channel
-        - eofs
-        - esmpy
+        - eofs>=1.4
+        - esmpy>=8.0.1
         - esmvalcore>=2.2.0,<2.3  # in esmvalgroup channel
-        - fiona
-        - gdal
+        - fiona>=1.8.18
+        - gdal>=3.1.4
         - iris>=3.0.1
-        - jinja2
-        - joblib
+        - jinja2>=2.11
+        - joblib>=1.0
         - lime
         - matplotlib>3.3.1  # bug in 3.3.1
-        - mpich<3.4  # avoid 3.4.1 external_2 from conda-forge like the plague
-        - natsort
+        # - mpich<3.4  # avoid 3.4.1 external_2 from conda-forge like the plague; stable 3.3.2 OK
```

Comment:
Remove?

Reply:
same, let's just be sure 3.4.1-external_2 is not causing us headaches again